src/components/shared/booleanDropdown.tsx

```diff
@@ -0,0 +1,19 @@
+import Dropdown from 'react-dropdown';
+import '../../styles/booleanDropdown.scss';
+
+const options = ['True', 'False'];
+
+export enum Position {
+  Top = 'top',
+  Middle = 'middle',
+  Bottom = 'bottom'
+}
+
+interface BooleanDropdownProps {
+  change: () => void;
```

Comment:
`on<Whatever>` is pretty standard for this!! It's to make it obvious in the naming that it's a method that is run when the drop down changes.
```suggestion
  onChange: () => void;
```